packages/core/upload/admin/src/components/EditAssetDialog/index.js

```diff
@@ -225,6 +225,14 @@ export const EditAssetDialog = ({
                           }),
                           value: getFileExtension(asset.ext),
                         },
+
+                        {
+                          label: formatMessage({
+                            id: getTrad('modal.file-details.id'),
+                            defaultMessage: 'File ID',
```

Comment:
In the admin app we call files `Assets`. Should we rename this to `Asset ID`?

Reply:
Yeah, it is an inconsistency: the backend calls them files and the frontend assets ... 🤦🏼